### PR TITLE
fix(examples): change colors to the current box design language

### PR DESCRIPTION
**Pull request by Sunyang730**
Screen Shot 2019-06-16 at 1 24 02 PM

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @Sunyang730 has signed the CLA. Thanks for the pull request!

**Comment by Sunyang730** [comment]
Screen Shot 2019-06-15 at 12 01 43 PM

**Comment by booc0mtaco** [comment]
Very nice! Looks like we have a duplicate: `Dark-Blue` and `Secondary-Blue`. I think we want the former for naming consistency. I also have some preliminary notes from design on a basic color/category mapping:

- `Green-Light` = Success
- `Watermelon-Red` = Error
- `Yellorange` = Warning
- `Yellow` = Highlight
- `Light-Blue` = Information
- `Purple-Rain` = Help

To improve density, we could create cards for each color category, and introduce a bit of text describing this mapping. 

```
+-------------------------------+
| Color w/ category description |
|-------------------------------|
| swatches in a column          |
+-------------------------------+
```

that way we can `inline-block` the sets so that it's responsive and easier to scan

**Comment by Sunyang730** [comment]
Screen Shot 2019-06-15 at 9 35 23 PM

**Comment by Sunyang730** [comment]
Screen Shot 2019-06-16 at 1 24 02 PM

File: examples/colors.md [hunk 1]
```diff
@@ -42,5 +42,23 @@ Object.keys(colors).forEach(colorKey => {
...
+    <h4>White</h4>
+    <div className="palette">
+      <div className="swatch-container">
+        <div className="swatch" style={{ backgroundColor: '#fff' }} />
+        <label>#fff - $white</label>
+      </div>
+    </div>
+  </div>
+  <div>
+    <h4>black</h4>
```
Comment (priyajeet) [inline examples/colors.md]: B

**Pull request merged**